lib/src/widgets/overlay.dart

```diff
@@ -689,36 +695,35 @@ class _Background extends StatelessWidget {
   final double transitionProgress;
   final Color color;
   final OverflowMode overflowMode;
-
-  const _Background({
-    Key key,
-    @required this.color,
-    @required this.state,
-    @required this.transitionProgress,
-    @required this.overflowMode,
-  })  : assert(color != null),
+  final double defaultOpacity;
```

Comment:
Why is this named `defaultOpacity` while it is just the opacity that were provided? (i.e. why haven't you just called it `opacity`?)

Reply:
Oh yeah that's right, there is already a getter named like that.
It's probably fine, even though a little bit ambiguous 😶